src/Backend/Core/Js/Components/Forms.js

```diff
@@ -76,7 +76,9 @@ export class Forms {
       const invalidTabId = $invalidField.closest('.tab-pane').attr('id')
 
       // Find the link that corresponds to the pane and have it show
-      $('a[href=#' + invalidTabId + '], [data-bs-target=#' + invalidTabId + ']').tab('show')
+      const invalidTab = document.querySelectorAll('a[href="#' + invalidTabId + '"][data-bs-toggle="tab"]')[0]
+      const invalidTabObject = new bootstrap.Tab(invalidTab)
+      invalidTabObject.show()
```

Comment:
Waarom hier eigenlijk niet starten met $?
Ik dacht dat de semi-ongeschreven regel zowat was: als het een element is doen variablename met $, anders niet.

Reply:
was dat niet enkel voor jQuery elementen? dit is gewoon een js object?

Reply:
Besproken: enkel jQuery elementen met $